src/main/java/org/neo4j/junit/jupiter/causal_cluster/CausalClusterStarter.java

```diff
@@ -0,0 +1,90 @@
+package org.neo4j.junit.jupiter.causal_cluster;
+
+import static java.util.stream.Collectors.*;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+
+import org.testcontainers.containers.Neo4jContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.SocatContainer;
+
+public class CausalClusterStarter {
```

Comment:
Can I have this package private and final please? 

```suggestion
final class CausalClusterStarter {
```